rclrs/Cargo.toml

```diff
@@ -33,7 +33,7 @@ futures-lite = { version = "2.6", features = ["std", "race"] }
 async-std = "1.13"
 
 # Needed for dynamic messages
-libloading = { version = "0.8", optional = true }
+libloading = "0.8"
```

Comment:
Argh, libloading 0.9 just got released a few weeks ago. Anyway, I think it's fine to keep it on 0.8 and upgrade to 0.9 in a follow-up branch, unless you want to give it a try @luca-della-vedova